source/globalCommands.py

```diff
@@ -3670,6 +3670,14 @@ def script_reportLinkDestination(
 		ti.expand(textInfos.UNIT_CHARACTER)
 		obj: NVDAObject = ti.NVDAObjectAtStart
 		presses = scriptHandler.getLastScriptRepeatCount()
+		if (
+			obj.role == controlTypes.role.Role.GRAPHIC  # If it's a graphic, and

```

Comment:
This comment is not very helpful, it just restates the code it is commenting

```suggestion
			obj.role == controlTypes.role.Role.GRAPHIC
```